### ISSUE TITLE
Patient/Doctor records upload

### ISSUE THREAD

**Issue by ARMmaster17** [CS4850Group5A/VirtuDoc#147]
# Summary
From the original project specification, and further input from the "customer", both the patient and the doctor users need the ability to upload files to the records pages.

Blocks #136
Blocked by #26 and #57

# Implementation
- [ ] Implement POST route and set as target of HTML form on records page.
- [ ] Create a FileEntity and upload to storage provider.
- [ ] Populate records page with list of all FileEntity instances in the DB (don't worry about filtering by who uploaded what. ACL support will be a separate ticket).
